geode-junit/src/main/java/org/apache/geode/test/junit/rules/serializable/MethodInvocationUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.test.junit.rules.serializable;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * Provides support for invocation of private methods by reflection.
+ */
+public class MethodInvocationUtils {
+
+  protected MethodInvocationUtils() {
+    // nothing
+  }
+
+  public static void invokeMethod(final Class targetClass, final Object targetInstance,
```

Comment:
Why pass in "targetClass" as a parameter? Couldn't you instead call "targetInstance.getClass"?
Should you call it "invokePrivateMethod"? We really don't want anyone to use this unless the target method is not accessible.

Reply:
Resolved